GVFS/GVFS.UnitTests/Prefetch/DiffHelperTests.cs

```diff
@@ -11,25 +11,26 @@
 
 namespace GVFS.UnitTests.Prefetch
 {
-    [TestFixtureSource(TestRunners)]
+    [TestFixtureSource(TestSymLinkSupport)]
     public class DiffHelperTests
     {
-        public const string TestRunners = "Runners";
-
-        private static readonly bool[] SymLinkSupport = new bool[]
-        {
-            true,
-            false,
-        };
-
+        public const string TestSymLinkSupport = "ShouldTestSymLinkSupport";
+
         public DiffHelperTests(bool symLinkSupport)
         {
             this.IncludeSymLinks = symLinkSupport;
         }
 
-        public static bool[] Runners
+        public static object[] ShouldTestSymLinkSupport

```

Comment:
Take a look at https://github.com/Microsoft/VFSForGit/pull/408/files#diff-118de5153571e4324419af8ab99e61a5R24, @wilbaker (and maybe other places in the codebase too) are being forced to repeat this same pattern.

If you would, please create a common property somewhere that we can use any time we want a test source with both bool values. We shouldn't have to repeat this boilerplate every time we need that.